File: Python/2.Python_Practice/1.Python/fabonacii.py



def fab():
    a=int(input("Enter a Number:"))

    b=0
    c=1

    for i  in range (2,a+1):
        su=b+c
        b=c
        c=su
        print(c)


# fab()



def fabb():
    a=int(input("Enter a Number:"))

    f=[0,1]

    for i in range (2,a+1):
        f.append(f[i-1]+f[i-2])
        print(f[i])

fabb()

File: 1-Coding/Practice/fabonacii.c
#include <stdio.h>

int fab(int a)
{
    int sum = 0;
    int i = 1;
    int j = 1;
    printf("1\t1\t");
    while (a >= i)
    {
        sum = j + i;
        i++;
        i = j;
        j = sum;
        printf("%d\t", sum);
    }
}

int main()
{
    int a;
    printf("Enter a number \n ");
    scanf("%d", &a);

    fab(a);
}